src/vivarium/framework/lookup/interpolation.py

```diff
@@ -109,7 +109,9 @@ def __call__(self, interpolants: pd.DataFrame) -> pd.DataFrame:
             )
 
         if self.categorical_parameters:
-            sub_tables = interpolants.groupby(list(self.categorical_parameters))
+            sub_tables = interpolants.groupby(
+                list(self.categorical_parameters), observed=False
```

Comment:
This just got rid of a bunch of FutureWarnings (`observed` is changing from default False to default True)

Reply:
```
observed: bool, default False
    This only applies if any of the groupers are Categoricals. If True: only show observed values for
    categorical groupers. If False: show all values for categorical groupers.

    Deprecated since version 2.1.0: The default value will change to True in a future version of pandas.
```